contrib/node/examples/src/node/web-component-button/BUILD

```diff
@@ -29,3 +29,32 @@ node_bundle(
   name='web-component-button-bundle',
   node_module=':web-component-button'
 )
+
+# The following target invokes webpack build.
```

Comment:
...because of the `build_script` argument? More comment would be helpful probably.